frontend/lib/ui/Feedback/Feedback.types.tsx

```diff
@@ -0,0 +1,276 @@
+        Je hebt bij <Link to={`../numbers`}>Aantal kiezers en stemmers</Link> ingevuld dat er minder
+        stemmen dan kiezers waren. Het aantal dat je bij J hebt ingevuld is niet gelijk aan het
+        aantal minder getelde stembiljetten.
+        <br />
+        Check of je het papieren proces-verbaal goed hebt overgenomen.
+      </div>
+    ),
+  },
+  F304: {
+    title: "Controleer I (stembiljetten meer geteld)",
+    code: "F.304",
+    content: (
+      <div>
+        Je hebt bij <Link to={`../numbers`}>Aantal kiezers en stemmers</Link> ingevuld dat er minder
+        stemmen dan kiezers waren. Daarom mag I niet ingevuld zijn.
+        <br />
+        Check of je het papieren proces-verbaal goed hebt overgenomen.
+      </div>
+    ),
+  },
+  F305: {
+    title: "Controleer ingevulde verschillen",
+    code: "F.305",
+    content: (
+      <div>
+        Je hebt bij <Link to={`../numbers`}>Aantal kiezers en stemmers</Link> ingevuld dat er
+        evenveel stemmen als kiezers waren. Maar je hebt wel verschillen ingevuld.
+        <br />
+        Check of je het papieren proces-verbaal goed hebt overgenomen.
+      </div>
+    ),
+  },
+  F401: {
+    title: "Controleer ingevoerde aantallen",
+    code: "F.401",
+    content: (
+      <div>
+        De opgetelde stemmen op de kandidaten en het ingevoerde totaal zijn niet gelijk.
+        <br />
+        Check of je het papieren proces-verbaal goed hebt overgenomen.
+      </div>
+    ),
+  },
+  W201: {
+    title: "Controleer aantal blanco stemmen",
+    code: "W.201",
+    content: (
+      <div>
+        Het aantal blanco stemmen is erg hoog.
+        <br />
+        Check of je het papieren proces-verbaal goed hebt overgenomen.
+      </div>
+    ),
+  },
+  W202: {
+    title: "Controleer aantal ongeldige stemmen",
+    code: "W.202",
+    content: (
+      <div>
+        Het aantal ongeldige stemmen is erg hoog.
+        <br />
+        Check of je het papieren proces-verbaal goed hebt overgenomen.
+      </div>
+    ),
+  },
+  W203: {
+    title: "Controleer aantal toegelaten kiezers en aantal uitgebrachte stemmen",
+    code: "W.203",
+    content: (
+      <div>
+        Er is een onverwacht verschil tussen het aantal kiezers (A t/m D) en het aantal getelde
+        stemmen (E t/m H).
+        <br />
+        Check of je het papieren proces-verbaal goed hebt overgenomen.
+      </div>
+    ),
+  },
+  W204: {
+    title: "Controleer herteld aantal toegelaten kiezers en aantal uitgebrachte stemmen",
+    code: "W.204",
+    content: (
+      <div>
+        Er is een onverwacht verschil tussen het herteld aantal kiezers (A.2 t/m D.2) en het aantal
+        getelde stemmen (E t/m H).
+        <br />
+        Check of je het papieren proces-verbaal goed hebt overgenomen.
+      </div>
+    ),
+  },
+  W205: {
+    title: "Controleer aantal uitgebrachte stemmen",
+    code: "W.205",
+    content: (
+      <div>
+        Het totaal aantal uitgebrachte stemmen (H) is 0.
```

Comment:
@jorisleker With the current font this `0` looks too much like an "O" to me:

![image](https://github.com/user-attachments/assets/f7690549-4575-4950-ab7e-f55b9a1843f3)

Reply:
Agreed. This is DM Sans, which is the brand font of Kiesraad. Just looked into it, and sadly it does not have an alternative zero with a dash or a dot. Think we'll have to live with this. Alternative would be to rephrase the sentence and say 'Het totaal aantal uitgebrachte stemmen (H) is nul. @Lionqueen94 WDYT?

Reply:
Or render the number `0` as Geist Mono?

Reply:
I think rephrasing it is best, I'll create a PR on the document

Reply:
@praseodym Not a big fan of combining different font-faces in one sentence. Different weights, spacing, etc. This is not an error that will be shown frequently, and there is no field marked O on this page. So the chance users won't understand what's said here is small. Let's just keep this as is.